src/main/kotlin/com/exactpro/th2/lwdataprovider/entities/requests/SseMessageSearchRequest.kt

```diff
@@ -28,19 +28,24 @@ import com.exactpro.th2.lwdataprovider.grpc.toProviderRelation
 import com.exactpro.th2.lwdataprovider.grpc.toStoredMessageId
 import java.time.Instant
 
-data class SseMessageSearchRequest(
+class SseMessageSearchRequest(
```

Comment:
Please, add `toString` method. Also, lets keep `startTimestamp` and `endTimestamp` closer to each other in `toString` method